src/docfx/build/legacy/LegacyUtility.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics;
+using System.IO;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class LegacyUtility
```

Comment:
we already have an `LegacyExtensions`, move the methods there to `LegacyUtility` and maybe make them non-extension methods?

Reply:
Merged these into LegacyUtility